README.md

```diff
@@ -1939,8 +1976,8 @@ Other Style Guides
     }
     ```
 
-  <a name="comments--todo"></a><a name="17.5"></a>
-  - [17.5](#comments--todo) Use `// TODO:` to annotate solutions to problems.
+  <a name="comments--todo"></a><a name="17.6"></a>
```

Comment:
these numbers are legacy, and must remain the same even if the sections are resorted.